src/test-helpers/getThoughtFromDB.ts

```diff
@@ -0,0 +1,10 @@
+import * as db from '../data-providers/dexie'
+import dbGetThought from '../data-providers/data-helpers/getThought'
+
+/**
+ * Get thought from db for given thought value.
+ */
+const getThoughtFromDB = async (thought: string) => {
```

Comment:
Let's call this `getLexemeFromDB`, otherwise it could be incorrectly assumed to get a `Child`.